components/lsp/LSP.tsx

```diff
@@ -95,45 +95,29 @@ const Testing = () => {
       contract.getCurrentTime().then((ts: ethers.BigNumber) => {
         setCurrentTime(ts.toString());
       });
+      const erc20 = createERC20ContractInstance(signer, data.collateralToken);
```

Comment:
Its a bit confusing to call this erc20 since the lsp has 3 of them. Might want to name it collateral or something like that

Reply:
Fair, renamed this.